### ISSUE TITLE
GPU and Batch Size

### ISSUE THREAD

**Issue by krbuettner** [wusize/ovdet#11]
Hello I was wondering if there is information on the GPU count, batch size, and GPU type for the results reported in the paper. Thanks!

**Comment by wusize** [comment]
By defaul we used 8 GPUs and set per-gpu batchsize as 2 for all experiments in the paper. However, when implementing the detector on MMDet3.x for the code release, we encounted issue on the training speed when using SyncBN. We sidestep this issue by using more GPUS (16), adjust the learning rate following the linear scaling rule, and train half of the original iterations.

For the configs that are not using SyncBN, we maintain the default setting of 8 GPUs.

**Comment by krbuettner** [comment]
Thank you for the response. I am asking because I plan to run on fewer GPUs (4) and may need to change the batch size in the codebase. Do you know where I can adjust per-GPU batch size? By default, would the per-gpu batch size just remain 2 if using fewer GPUs?

**Comment by wusize** [comment]
Hi! Please refer to this [config file](https://github.com/wusize/ovdet/blob/2d8ad51cc32512c5da6f2e11302838ddb48f465e/configs/_base_/datasets/coco_ovd_kd.py#L45).

**Comment by wusize** [comment]
And increase the per-gpu batch size when you use fewer gpus. Otherwise, linearly adjust the learning rate and increase the total number of iterations.

**Comment by krbuettner** [comment]
Great, thank you for the guidance.

**Comment by kunpeng337** [comment]
Hello, I encountered some problems in the process of deploying code, such as not finding configs and checkpoints when running test.py, and their positions (as shown below). I hope you can help me. My email is limi.1232321@gmail.com. And qq email 120001098@qq.com
![7O(5EZB(A{%PV()ZHXQ1$JE](https://github.com/user-attachments/assets/7744b18a-8128-44f3-be2c-05215b592410)
![OS16TPHX79JO%XXUBZK B88](https://github.com/user-attachments/assets/a316337e-f493-4ba9-bac9-20692ed2054a)
